util/gen_coverage_report.ts

```diff
@@ -508,12 +489,41 @@ const writeMissingTranslations = (missing: MissingTranslations[], outputFileName
     if (isNaN(tagDateObj.getTime())) {
       tagDateObj = new Date(commitDate);
     }
-    tags[tagName] = {
+    unsortedTags.push({
+      name: tagName,
       tagDate: tagDateObj.getTime(),
       tagHash: tagHash,
+      files: [],
+    });
+  }
+
+  const tags: Tags = {};
+
+  // This hash is the default "initial commit" hash for git, all repos have it
+  let lastVersion = '4b825dc642cb6eb9a060e54bf8d69288fbee4904';
```

Comment:
Should this live near the top of the file for visibility?

Reply:
I guess there could be a `const DEFAULT_COMMIT_HASH = '4b825dc642cb6eb9a060e54bf8d69288fbee4904';` at the top, then 503 could change to `  let lastVersion = DEFAULT_COMMIT_HASH;`?